ledger/src/blockstore_db.rs

```diff
@@ -1227,6 +1245,39 @@ impl TypedColumn for columns::OptimisticSlots {
     type Type = blockstore_meta::OptimisticSlotMetaVersioned;
 }
 
+impl Column for columns::MerkleRootMeta {
+    type Index = (Slot, u64);
```

Comment:
Can we add a comment what `u64` is? Something like:
```rust
type Index = (Slot, /*fec_set_index:*/ u64);
```

also `fec_set_index` is `u32`:
https://github.com/solana-labs/solana/blob/a96be5d2f/ledger/src/shred.rs#L208
Can we use `u32` for consistency?

Reply:
changed it to `u32`, not sure why we are using `u64` everywhere, might have been an artifact of the original pr https://github.com/solana-labs/solana/pull/5461/files